middleware/src/main/java/com/anytypeio/anytype/middleware/interactor/Middleware.kt

```diff
@@ -965,9 +965,7 @@ class Middleware @Inject constructor(
         if (BuildConfig.DEBUG) logRequest(request)
         val response = service.objectCreateObjectType(request)
         if (BuildConfig.DEBUG) logResponse(response)
-        return ObjectWrapper.Type(
-            response.details ?: throw IllegalStateException("Missing details")
-        )
+        return response.details
```

Comment:
`response.details.validateStruct() ?: throw IllegalStateException("Invalid object type struct")`

Reply:
What is validateStruct() method? We don't have such.
Chnaged to `return response.details ?: throw IllegalStateException("Null object type struct")`